Framework/DataHandling/test/SetSampleMaterialTest.h

```diff
@@ -261,7 +261,7 @@ class SetSampleMaterialTest : public CxxTest::TestSuite {
     TS_ASSERT_DELTA(sampleMaterial.absorbXSection(NeutronAtom::ReferenceLambda),
                     4.49, 0.0001);
     TS_ASSERT_DELTA(
-        sampleMaterial.cohScatterLength(NeutronAtom::ReferenceLambda), 10.2877,
+        sampleMaterial.cohScatterLength(NeutronAtom::ReferenceLambda), 10.3,
```

Comment:
The values 4.816 and 10.3 appear to be magic numbers. Perhaps they could be replaced by constants with meaningful names.

Reply:
They are magic numbers. Since they're in unit tests, I'm not sure that they would really benefit from being made constants.